binja/README.md

```diff
@@ -0,0 +1,30 @@
+# Binary Ninja KRF Analysis
+This folder contains three distinct sections of python scripts.
+
+## Extraction set
+First, a set to be run in the environment where the fuzzing occured in order to extract the binaries and coredumps and put them in a tarball.  
+These are `main.py` and `gdb.py`. They must be in the same directory when `main.py` is run. 
+
+### Use
+```bash
+# Variable number of coredumps
+python3 main.py path/to/executable path/to/coredump1 [path/to/coredump2] ...
+```
+
+This will internally execute `gdb` on the coredump using `gdb.py` to extract the neccessary data, then create a tarball named `krfanalysis.tar.gz`.
+The directory `krfanalysis` must not exist before the tool is run, or it will fail.
+
+## Analysis set
+Second, an analysis set is run on a computer with binary ninja installed and the `binaryninja` python module in the python path.
```

Comment:
It'd be nice to add a little snippet here for putting binja on the `PYTHONPATH`, at least for macOS.